cumulus/tasks/discover-pdrs/index.js

```diff
@@ -24,6 +24,7 @@ function discoverPdrs(event) {
     const templateUri = config.templateUri;
     const collection = config.collection;
     const provider = config.provider;
+    // FIXME Can config.folder not be used?
```

Comment:
Is this resolved? It seems like setting `config.folder` and then passing it to the `Discover` constructor should work.

Sidenote: that option used to be called `pdrFolder`, not sure if there's still a reference to that somewhere, but I didn't see it.

Reply:
It is not resolved, I just came across it as I was writing tests.  I'll defer to @scisco as to whether there should be a ticket written for it or not.

Reply:
let's leave it as is for now.